streamer/output_stream.py

```diff
@@ -60,16 +59,44 @@ def is_dash_only(self) -> bool:
     """Returns True if the output format is restricted to DASH protocol"""
     assert self.codec is not None
     return self.codec.get_output_format() is 'webm'
+  
+  def get_init_seg_file(self, **kwargs) -> Pipe:
+    INIT_SEGMENT = {
+      MediaType.AUDIO: '{dir}/audio_{language}_{channels}c_{bitrate}_{codec}_init.{format}',
+      MediaType.VIDEO: '{dir}/video_{resolution_name}_{bitrate}_{codec}_init.{format}',
+      MediaType.TEXT: '{dir}/text_{language}_init.{format}',
+    }
+    path_templ = INIT_SEGMENT[self.type].format(**self._features, **kwargs)
```

Comment:
Nit: I think I would prefer the complete words, for clarity: `path_template`

But then I look again and realize it's not a template, it's an actual path.  The template was the value from the dictionary, but after format(), you get a concrete path.  So how about `path`?